charts/fluent-operator/values.yaml

```diff
@@ -0,0 +1,182 @@
+# Default values for fluentbit-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Set this to containerd or crio if you want to collect CRI format logs
+containerRuntime: docker
+# You can also deploy the Kubernetes logging stack with Helm, just need to set the Kubernetes parameter to ture.
```

Comment:
I'm not entirely certain what this means - this is a Helm chart so surely we are already deploying it with Helm?

Also typo on "ture" -- should be "true"

Reply:
Agree,calling it `Kubernetes` is confusing, we can explain like this:
`# You can also deploy the Kubernetes logging stack with Helm, just need to set the Kubernetes parameter to ture.`
=>
# If you want to deploy a default Fluent Bit pipeline (including Fluent Bit Input, Filter, and output) to collect Kubernetes logs, you'll need to set the Kubernetes parameter to true.

Reply:
Ah right, that makes sense to me now - so it's explicitly for getting Kubernetes logs.

Do we cope with some of the extra requirements on Openshift btw (e.g SCC and fs config)?

Reply:
OpenShift is not yet tested, but I assume the log format doesn't change much with K8s

Reply:
Nope, it's more the setup around mounting the volumes: you need to be able to mount host volumes and with the right security set up. 
Shameless plug: https://github.com/calyptia/openshift-fluent-bit-examples
The helm chart covers it pretty well now with a simple toggle I think.